dispatch/static/manager/src/js/reducers/ToasterReducer.js

```diff
@@ -184,6 +184,15 @@ export default function toasterReducer(toaster = {}, action) {
     return showToast(`${action.payload.length} event${action.payload.length > 1 ? 's' : ''} deleted`)
   case rejected(types.EVENTS.DELETE_MANY):
     return showToast('Some events could not be deleted', Intent.DANGER)
+  case fulfilled(types.EVENTS.PUBLISH):
```

Comment:
these publish actions are never fired off, and so these toast messages are never used.